docker/1.4.0/final/Dockerfile.gpu.training

```diff
@@ -0,0 +1,50 @@
+ARG py_version
+
+FROM nvidia/cuda:9.2-cudnn7-runtime-ubuntu16.04
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+RUN apt-get update && apt-get install -y --no-install-recommends software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa -y && \
+    apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        libopencv-dev \
+        curl && \
+    if [ $py_version -eq 3 ]; then PYTHON_VERSION=python3.6; else PYTHON_VERSION=python2.7; fi && \
+        apt-get update && apt-get install -y --no-install-recommends $PYTHON_VERSION-dev && \
+        ln -s -f /usr/bin/$PYTHON_VERSION /usr/bin/python && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install pip
+RUN cd /tmp && \
+    curl -O https://bootstrap.pypa.io/get-pip.py && \
+    python get-pip.py 'pip==19.1' && rm get-pip.py
+
+WORKDIR /
+
+COPY dist/sagemaker_mxnet_container-2.0.0.tar.gz /sagemaker_mxnet_container-2.0.0.tar.gz
+
+RUN pip install --no-cache mxnet-cu92mkl==1.4.0.post0 \
```

Comment:
Hi, 1.4.0.post0 was a temporary fix to enable this package on cpu instances but it did not work as expected. We recommend to use 1.4.0 version of MXNet in all MXNet containers